pandas/conftest.py

```diff
@@ -1317,6 +1317,22 @@ def nullable_string_dtype(request):
     return request.param
 
 
+@pytest.fixture(
+    params=[
+        pytest.param("str[pyarrow]", marks=td.skip_if_no("pyarrow")),
```

Comment:
I was going to comment: I don't think this can work. Although it is then strange the tests are passing :) But it seems this was not doing what I think you expected it was doing -> https://github.com/pandas-dev/pandas/pull/60661

I would use the same approach of creating the dtype through StringDtype(..) like in some of the fixtures above